flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/functions/BuiltInAggregateFunctionTestBase.java

```diff
@@ -362,6 +387,70 @@ public void execute(TableEnvironment tEnv, Table sourceTable) {
         protected abstract TableResult getResult(TableEnvironment tEnv, Table sourceTable);
     }
 
+    private abstract static class ValidationErrorItem implements TestItem {
+        private final Class<? extends Throwable> exceptionClass;
+        private final String exceptionMessage;
+
+        public ValidationErrorItem(String exceptionMessage) {
+            this.exceptionClass = ValidationException.class;
```

Comment:
We can parameterize this `exceptionClass` and change `ValidationErrorItem` to `ErrorTestItem` for a more general purpose, otherwise a constant is enough.